### PR TITLE
Fix help messages

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Fix help messages on the `internal/pkg/cli`

the current help message has an extra "\t" in the last line, causing that the shell prompt be moved after a help call
Attn: @singularityware-admin

### CODE REVIEW

File: internal/pkg/cli/build.go [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("build called")
+	},
+}
+
+func init() {
+	singularityCmd.AddCommand(buildCmd)
+
+	// ultimately all of this should go into a seperate function to be shared
+	// between exec, run, and shell
```
Comment (GodloveD) [inline internal/pkg/cli/build.go]: This comment is not applicable to the build command.

**Review by GodloveD — CHANGES_REQUESTED** [review]
I think if you just remove those comments than you will be good to go.

**Comment by ArangoGutierrez** [comment]
sorry COPY/PASTE typo, fixed
Thanks @GodloveD

**Review by GodloveD — APPROVED** [review]

**Comment by GodloveD** [comment]
Looks good to me.  Ready to merge.  Are we doing 2 reviews per PR for 3.0 branch?

**Comment by bauerm97** [comment]
@GodloveD Not necessarily, but I'm the Golang gatekeeper so I want to do all the merging. Looks good ed, good start.

**Pull request merged**